README.md

```diff
@@ -89,6 +89,7 @@
 * [Nixery](https://github.com/tazjin/nixery) - A Docker-compatible container registry which builds images ad-hoc via Nix.
 * [Nixinate](https://github.com/MatthewCroughan/nixinate) - A Nix flake library to provide app outputs for managing existing NixOS hosts over SSH.
 * [NixOps](https://github.com/NixOS/nixops) - The official Nix deployment tool, compatible with AWS, Hetzner, and more.
+* [nixos-rebuild](https://www.haskellforall.com/2023/01/announcing-nixos-rebuild-new-deployment.html) - A simple method for managing NixOS hosts over SSH.
```

Comment:
So, I think this is a decent _guide_, that could probably go in the "learning" section at the top of the document, but it would need a different description and name there.

That being said, `nixos-rebuild` is definitely very powerful, but I can't find _any_ official documentation besides the manpage, and I would not consider <https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/manpages/nixos-rebuild.8> to be "readable" on the web, though there _really_ should be one